### PR TITLE
kelda: Rename binaries, user, and docker image

### PR DESCRIPTION
**Pull request by ejj**
This patch focuses on renaming some of the core system components --
specifically the binaries, docker container image, and the ubuntu
user name on the minions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,22 +30,22 @@ install:
...
+    - make -j 2 check lint coverage check-blueprints docker-build-kelda
     - (cd integration-tester && go build . && make tests)
 
 after_success:
     - bash <(curl -s https://codecov.io/bash)
     - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
           docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
           if [ "$TRAVIS_BRANCH" == "master" ]; then
-              docker push quilt/quilt:latest;
+              docker push kelda/kelda:latest;
```
Comment (kklin) [inline .travis.yml]: keldaio/kelda, right?

File: .travis.yml [hunk 2]
```diff
@@ -30,22 +30,22 @@ install:
...
               git -c user.email="builds@travis-ci.com" -c user.name="Travis CI"
                 tag -fa dev -m "Autogenerated tag to track master branch";
-              git push -fq https://$GITHUB_KEY@github.com/quilt/quilt dev;
+              git push -fq https://$GITHUB_KEY@github.com/kelda/kelda dev;
           fi;
           if [ "$TRAVIS_TAG" != "" ] ; then
-              docker tag quilt/quilt quilt/quilt:$TRAVIS_TAG;
-              docker push quilt/quilt:$TRAVIS_TAG;
+              docker tag kelda/kelda kelda/kelda:$TRAVIS_TAG;
```
Comment (kklin) [inline .travis.yml]: Same here (and the line below) re keldaio/kelda

File: .travis.yml (hunk 1 above)
Comment (ejj) [reply .travis.yml]: oops good catch

**Comment by ejj** [comment]
@kevin w.r.t. your comments:
```
debug-logs shells out to Quilt (Kelda), so there's a reference there that needs to be replaced.```
I'll fix in this patch.

```
Some error messages that instruct users to run quilt commands (such as quilt daemon) are now out of date```
So much stuff is out of date =).  I think in this patch I just want to focus on getting things working, and then I'll have a future patch where I fix up the aesthetic stuff.

```
The integration-tester shells out to the Quilt (Kelda) binary```
Yep I'm aware.  I'd like to merge this first, and then next up my goal is to get integration tests working.  Since the tests are already broken, it will be easier to just get this in while I hack on the tester.

File: .travis.yml [hunk 3]
```diff
@@ -30,22 +30,22 @@ install:
...
+              docker push keldaio/kelda:latest;
               git -c user.email="builds@travis-ci.com" -c user.name="Travis CI"
                 tag -fa dev -m "Autogenerated tag to track master branch";
-              git push -fq https://$GITHUB_KEY@github.com/quilt/quilt dev;
+              git push -fq https://$GITHUB_KEY@github.com/kelda/kelda dev;
           fi;
           if [ "$TRAVIS_TAG" != "" ] ; then
-              docker tag quilt/quilt quilt/quilt:$TRAVIS_TAG;
-              docker push quilt/quilt:$TRAVIS_TAG;
+              docker tag keldaio/kelda kelda/kelda:$TRAVIS_TAG;
```
Comment (kklin) [inline .travis.yml]: One more keldaio reference

**Pull request merged**